downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -113,19 +113,75 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
     ```
 
 #### OpenAPI tool
-
+-  Added support to generate a mock client for OpenAPI specifications (OAS) that include examples. To generate a mock client, use the `--mock` flag in the OpenAPI client generation CLI command.
+  ```
+  $ bal openapi -i <yml file> --mode client --mock
+  ```
+-  Provided an option to generate a single Ballerina file with the client or the service code. To generate code in a single file, use the `--single-file` flag in the OpenAPI client/service generation CLI command.
+  ```
+  $ bal openapi -i <yml file> --mode <client|service> --single-file
+  ```
+-  Added support to generate the Ballerina service contract object type for a given OAS. To generate the service contract object type, use the `--with-service-contract` flag along with the OpenAPI CLI client/service generation command.
+  ```
+  $ bal openapi -i <yml file> --mode <client|service> --with-service-contract
+  ```
+-  Introduced example annotations in the OpenAPI package. This feature will  enable rendering example schemas in the generated OpenAPI specification.
+ex:
+  1. Using the `openapi:Example` annotation
+ ```ballerina
+    @openapi:Example {
+      value: {
+        id: 10,
+        name: "Jessica Smith"
+      }
+    }
+    type User record {
+      int id;
+      string name
+    }
+ ```
+  2. Using the `openapi:Examples` annotation
+  ```ballerina
+    @openapi:Examples {
+      Jessica: { // Example 1
+        value: {
+           id: 10,
+           name: "Jessica Smith"
+        }
+      },
+      Ron: { // Example 2
+        value: {
+           id: 11,
+           name: "Ron Stewart"
+        }
+      } 
+    }
+    type User record {
+      int id;
+      string name
+    }
+  ```
+- Provided a flag to generate the Ballerina client/service adhering to Ballerina naming conventions. To enable this feature, use the `--use-sanitized-oas` flag in the OpenAPI client/service CLI command. This an experimental feature.
```

Comment:
Was `sanitized-oas` the correct term here? Aren't we sanitizing the generated code?

Reply:
Yeah, this flag modifies the given YAML according to Ballerina naming conventions. Currently, we generate the client/service based on the user-provided YAML without engaging any Ballerina naming conventions.